ggml/src/ggml-sycl.cpp

```diff
@@ -3658,6 +3658,10 @@ static void ggml_sycl_mul_mat(ggml_backend_sycl_context & ctx, const ggml_tensor
     use_mul_mat_q = use_mul_mat_q && (src1->ne[1] <= MMQ_MAX_BATCH_SIZE);
 #endif // SYCL_USE_XMX
 
+    // mmvq path is faster in the Nvidia backend but slower on the Intel backend
```

Comment:
// mmvq path is faster in the Nvidia backend but slower on the Intel backend

Suggest removing this comment:
1. No need such detailed comment for 2 lines codes. 
It's common sense to optimize by adding branch for hardware type.

2. We should avoid mention the brand of commercial company in code.
Suggest using CUDA and SYCL for them if needed.

Reply:
I disagree with 1. I agree that it makes sense to optimize for hardware type, but the reasoning and the change in the dispatch is not necessarily obvious to anyone that hasn't been actively developing the backend. 

I agree with 2.